spec/controllers/proposals_controller_spec.rb

```diff
@@ -54,16 +54,16 @@
       it 'should allow the requester to see it' do
         proposal = create(:proposal, requester: user)
         get :show, id: proposal.id
+        expect(response.status).to eq(200)
         expect(response).not_to redirect_to("/proposals/")
         expect(flash[:alert]).not_to be_present
       end
 
       # might be flaky? -Aidan, 8/14/15
```

Comment:
remove this comment while we're here? (not sure what flaky means in this context)